Tools/clinic/clinic.py

```diff
@@ -2490,7 +2513,7 @@ def parse(self, input: str) -> str:
 
                     block.input = 'preserve\n'
                     printer_2 = BlockPrinter(self.language)
-                    printer_2.print_block(block, core_includes=True)
+                    printer_2.print_block(block, core_includes=True, clinic=self)
```

Comment:
See comment above about potentially just passing ``clinic.includes``:
```suggestion
                    printer_2.print_block(block, core_includes=True, clinic=self.includes)
```